tests/alerts_tests.py

```diff
@@ -41,114 +49,267 @@
 def setup_database():
     with app.app_context():
         slice_id = db.session.query(Slice).all()[0].id
-        database_id = utils.get_example_database().id
+        example_database = utils.get_example_database()
+        example_database_id = example_database.id
+        example_database.get_sqla_engine().execute(
+            "CREATE TABLE test_table AS SELECT 1 as first, 2 as second"
+        )
+        example_database.get_sqla_engine().execute(
+            "INSERT INTO test_table (first, second) VALUES (3, 4)"
+        )
 
+        common_data = dict(
+            active=True,
+            crontab="* * * * *",
+            slice_id=slice_id,
+            recipients="recipient1@superset.com",
+            slack_channel="#test_channel",
+        )
         alerts = [
-            Alert(
-                id=1,
-                label="alert_1",
-                active=True,
-                crontab="*/1 * * * *",
+            Alert(**common_data, label="alert_1"),
```

Comment:
it seems like there are too many alerts here, try to keep bare minimum for the needed ones for the test e.g. reuse alerts / observers. 
maybe create a utility function that will take alerts name, config and observer sql and will create a set of objects for you etc